bubblewrap.c

```diff
@@ -1869,7 +1869,7 @@ main (int    argc,
   acquire_privs ();
 
   /* Never gain any more privs during exec */
-  if (prctl (PR_SET_NO_NEW_PRIVS, 1, 0, 0, 0) < 0)
+  if (is_privileged && prctl (PR_SET_NO_NEW_PRIVS, 1, 0, 0, 0) < 0)
```

Comment:
The security of bubblewrap completely depends on PR_SET_NO_NEW_PRIVS. It is fundamentally what makes it different from e.g. chroot(), as you can't fool setuid binaries with a modified filesytem view. 

I'm surprised this ever worked at all. seccomp filters require this to be set, but I thought user namespaces did too.